alarm/nialarm.yml

```diff
@@ -0,0 +1,686 @@
+          schema:
+            type: object
+            title: Request body
+            required:
+              - transition
+              - alarmId
+            properties:
+              alarmId:
+                description: A value meant to uniquely identify the particular process or condition tracked by a given alarm. For example, alarms created by the Tag Rule Engine Service have their alarmIds set to the concatenation of the path of the tag which caused the rule to be triggered and the ID of the rule.
+                type: string
+                example: CRIO1.System.Health.DiskSpaceMeanUsePercentage.Bdd9u!4aMd!$pYrf*CnaIZ2tbu$-Ct%?
+              transition: 
+                $ref: '#/definitions/AlarmTransition'
+              createdBy:
+                description: String identifying who or what created a given alarm instance. This is usually used to identify the particular
+                  rule engine which requested that a given alarm be created.
+                type: string
+                example: TagRuleEngine
+              channel:
+                description: An identifier for the tag or resource associated with the alarm
+                type: string
+                example: CRIO1.System.Health.DiskSpaceMeanUsePercentage
+              resourceType:
+                description: The type of resource associated with the alarm
+                type: string
+                example: Tag
+              displayName:
+                description: Optional display name for a given alarm instance. Display names do not need to be unique.
+                type: string
+                example: Low disk space on tester-1.
+              description:
+                description: Optional description text for a given alarm instance
+                type: string
+                example: Disk space on tester-1 is high. Disk usage was 95% when this alarm was created.
+              keywords:
+                description: Keywords associated with an alarm instance. Alarm instances can be tagged with keywords to make it easier to find them with queries.
+                type: array
+                items:
+                  type: string
+                example:
+                  - keyword1
+                  - keyword2
+              properties:
+                description: Key-value-pair metadata associated with an alarm instance. Alarm instances can be tagged with properties so that they can be more easily found with queries. Additionally, alarm instance properties are used to expand message templates in the notification strategies associated with a given alarm instance.
+                type: object
+                example:
+                  key1: value1
+          required: true
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+            title: ID of the created or modified instance
```

Comment:
This seems like kind of an oddly specific title. Maybe it could be more symmetric with the request body, like `Create or update instance response`?

Reply:
Fixed the title